src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/ubuntu-dotnet-local-install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Argument = -d destdir
+usage()
+{
+cat << EOF
+usage: $0 options
+
+This script installs dotnet cli on the linux machine
+
+OPTIONS:
+   -h      Show this message
+   -d      directory where dotnet cli will be installed
+
+EOF
+}
+
+DESTDIR=
+while getopts “hd:” OPTION
+do
+     case $OPTION in
+         h)
+             usage
+             exit 1
+             ;;
+         d)
+             DESTDIR=$OPTARG
+             ;;
+         ?)
+             usage
+             exit
+             ;;
+     esac
+done
+
+echo $DESTDIR
+if [[ -z $DESTDIR ]]
+then
+     usage
+     exit 1
+fi
+
+echo "Initiating local dotnet install"
+wget https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.sh
+chmod 777 dotnet-install.sh
+wget https://raw.githubusercontent.com/Microsoft/xunit-performance/master/DotNetCliVersion.txt
+versionnum=$(cat DotNetCliVersion.txt)
+mkdir $DESTDIR
+source dotnet-install.sh -i $DESTDIR -v $versionnum
```

Comment:
The installation will be put the payload directory of that particular correlation id and should not affect the overall state of the machine, in other words this is a local or temporary installation which means when the correlation payload is cleared out the installation should automatically deleted